Inconsistencies between declared TypeScript export and actual export

The TypeScript declaration file declares `Speaker` as a `default` export:

https://github.com/TooTallNate/node-speaker/blob/f365fc8b84c0a856c1bdc71255da01d7aee6ac02/index.d.ts#L20-L27

If that was the case, `Speaker` could be imported with the following syntax:

```typescript
import Speaker from "speaker";
```

This does not work, however, since the actual file does not assign `exports.default`, but rather `exports`:

https://github.com/TooTallNate/node-speaker/blob/f365fc8b84c0a856c1bdc71255da01d7aee6ac02/index.js#L353-L357

The correct way to import `Speaker` would be to use

```typescript
import * as Speaker from "speaker";
```

...which yields the following (incorrect) error by the compiler though:

```
src/output/SpeakerOutput.ts:6:19 - error TS2709: Cannot use namespace
'Speaker' as a type.

6  private speaker: Speaker;
                    ~~~~~~~
9       this.speaker = new Speaker(options);
                       ~~~~~~~~~~~~~~~~~~~~
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,20 +1,22 @@
 import { Writable, WritableOptions } from 'stream';
 
-export interface Options extends WritableOptions {
-    readonly channels?: number;
-    readonly bitDepth?: number;
-    readonly sampleRate?: number;
-    readonly lowWaterMark?: number;
-    readonly highWaterMark?: number;
-}
+namespace Speaker {
+    interface Options extends WritableOptions {
+        readonly channels?: number;
+        readonly bitDepth?: number;
+        readonly sampleRate?: number;
+        readonly lowWaterMark?: number;
+        readonly highWaterMark?: number;
+    }
 
-export interface Format {
-    readonly float?: boolean;
-    readonly signed?: boolean;
-    readonly bitDepth?: number;
-    readonly channels?: number;
-    readonly sampleRate?: number;
-    readonly samplesPerFrame?: number;
+    interface Format {
+        readonly float?: boolean;
+        readonly signed?: boolean;
+        readonly bitDepth?: number;
+        readonly channels?: number;
+        readonly sampleRate?: number;
+        readonly samplesPerFrame?: number;
+    }
 }
 
 /**
@@ -23,8 +25,8 @@ export interface Format {
  *
  * @param opts options.
  */
-export default class Speaker extends Writable {
-    constructor(opts?: Options);
+class Speaker extends Writable {
+    constructor(opts?: Speaker.Options);
 
     /**
      * Closes the audio backend. Normally this function will be called automatically
@@ -42,7 +44,7 @@ export default class Speaker extends Writable {
      * @param format format object with `channels`, `sampleRate`, `bitDepth`, etc.
      * @return MPG123_ENC_* constant, or `null`
      */
-    public getFormat(format: Format): number | null;
+    public getFormat(format: Speaker.Format): number | null;
 
     /**
      * Returns whether or not "format" is playable via the "output module"
@@ -53,3 +55,5 @@ export default class Speaker extends Writable {
      */
     public isSupported(format: number): boolean;
 }
+
+exports = Speaker

